usaspending_api/database_scripts/matview_generator/summary_transaction_month_view.json

```diff
@@ -213,6 +213,10 @@
       "name": "type_set_aside",
       "where": "\"type_set_aside\" IS NOT NULL",
       "columns": [{"name": "\"type_set_aside\""}]
+    }, {
```

Comment:
The addition of the GIN index here was propagated to the SQL files below after the matview SQL generator was run to incorporate the change, is that a correct understanding?

Reply:
Correct. I hardcoded the "random" id to keep the PR small. For larger edits (like add/remove cols) the random index string should be re-applied